### ISSUE TITLE
Question regarding the shutdown function vs destruct's stop server

### ISSUE THREAD

**Issue by grantlucas** [tiger-seo/PhpBuiltinServer#19]
Out of curiosity, is it required to [register a shutdown function](https://github.com/tiger-seo/PhpBuiltinServer/blob/ec61122a0189312b24f409cfd1a610e35ecc403a/src/Codeception/Extension/PhpBuiltinServer.php#L42) to terminate the resource in addition to the `stopServer()` call in `__desctruct()`?

**Comment by tiger-seo** [comment]
yep, cause, AFAIK, it is not guaranteed to call ``__destruct()`` in any case

**Comment by grantlucas** [comment]
Thanks! Another question. Instead of calling just `proc_terminate()` in the shutdown function, would calling `stopServer()` make sense to ensure it all get shut down properly in each case?

I haven't experienced any issues, just wondering about the different shutdown approaches.
